cea/interfaces/dashboard/plots/static/js/dashboard-plots.js

```diff
@@ -70,20 +74,35 @@ function load_all_plots() {
         let x_table_id = "#x_table-" + dashboard_index + "-" + plot_index;
 
         $.get("../div/" + dashboard_index + "/" + plot_index, function(data){
```

Comment:
yeah, I'm sorry about that line: i _hate_ concatenating strings. that's why i've started using `"".format()` in python. at the time i wrote that, i didn't know about template strings yet - only now reading up on all the new features javascript has. it's come a long way since 1999!

Reply:
Will help to change the string concatenations to templates (though Internet Explorer is the only browser that doesn't support it, but shouldn't be an issue)

Yeah and there's so many javascript frameworks now too! Actually I have been trying to see if I can convert the front end to use **Electron** and **React** during my free time. Pretty cool stuff.

Reply:
yeah, I've been looking into electron myself lately and it _would_ solve the IE issue... but instead of React, I was looking into Vue.js - it seems React is to Vue what Django is to Flask?